src/content/docs/self-hosting/kubernetes.mdx

```diff
@@ -131,6 +157,21 @@ spec:
   selector:
     io.kompose.service: atuin
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: postgres
+  name: postgres
```

Comment:
This is the new service which is dns addressible from the atuin pod as `postgres`